src/bo4e/com/zaehlwerk.py

```diff
@@ -39,29 +42,29 @@ class Zaehlwerk(COM):
     zaehlwerk_id: Optional[str] = None  # Identifikation des Zählwerks (Registers) innerhalb des Zählers.
     # Oftmals eine laufende Nummer hinter der Zählernummer. Z.B. 47110815_1
     bezeichnung: Optional[str] = None  # Zusätzliche Bezeichnung, z.B. Zählwerk_Wirkarbeit.
-    richtung: Optional[Energierichtung] = None  # Die Energierichtung, Einspeisung oder Ausspeisung.
-    obis_kennzahl: Optional[
-        str
-    ] = None  # Die OBIS-Kennzahl für das Zählwerk, die festlegt, welche auf die gemessene Größe mit dem Stand gemeldet wird.
+    richtung: Optional["Energierichtung"] = None  # Die Energierichtung, Einspeisung oder Ausspeisung.
+    obis_kennzahl: Optional[str] = (
```

Comment:
bissle komisch formtiert, oder?

Reply:
So machte das Black bei mir aber auch neuerdings

Reply:
Ich würde sagen, besser als wenn er den Zeilenumbruch in der Typdefinition macht.